data/libs/Legal.lua

```diff
@@ -53,7 +46,7 @@ end
 
 function Legal:notifyOfCrime (ship, crime)
 	if not ship:IsPlayer() then return end
-	-- TODO: can this be called in hyperpace?
+	-- TODO: can this be called in hyperspace?
```

Comment:
:+1: 
Ha! Makes me think of Swedish Bathory's "Pace til death".

Reply:
:metal: